### ISSUE TITLE
Segmentation prediction does not display

### ISSUE THREAD

**Issue by kahminaford4** [AllenCell/allencell-segmenter-ml#531]
When selecting both segmentation models megaSeg and megaSeg lite and selecting an input image, the prediction runs for several minutes and then finishes with no segmentation prediction displayed or generated in the layer list. No errors or other indicators are shown.

The only noticable difference is that the raw image has [raw] prefaced and does not show up as an on-screen image.
<img width="857" alt="Screenshot 2024-10-15 140851" src="https://github.com/user-attachments/assets/65b09228-ff6f-4bbc-912a-0d4dbc688378">

**Comment by yrkim98** [comment]
Hi @kahminaford4  thanks for the bug report.
Could you help us diagnose this issue further?

if you go to the Output directory you selected in the plugin, do you see a `target` folder and any images in that folder?

Could you also run this again, go to the `Plugins` > `Plugin Errors` and copy and paste the full error log here as well?

thank you

**Comment by kahminaford4** [comment]
Hello - thanks for the help.

1. There are no images in the target folder.
2. When I go to this there is nothing to select: 
<img width="802" alt="image" src="https://github.com/user-attachments/assets/80e9995f-d20e-4798-aa91-88bded6ffaf4">
3. CPU
4. python 3.10.15
5. I installed from pip.

I figured out my issue - I ran out of memory while trying to run the segmentation after looking through my terminal. 

<img width="863" alt="image" src="https://github.com/user-attachments/assets/7e6b1ef6-2aa2-4d6e-9bcf-bd57d6b49d3a">

Is there anything I can do to help reduce/minimize memory usage with this tool?
